### PR TITLE
refactor(api-gen): improve deprecation message layout

### PR DESCRIPTION
**Pull request by JeanMeche**

### CODE REVIEW

File: bazel/api-gen/rendering/templates/class-method-info.tsx [hunk 1]
```diff
@@ -37,7 +37,6 @@ export function ClassMethodInfo(props: {
       ) : (
         <div>
           <DeprecatedLabel entry={entry} />
-          <span>{entry.deprecationMessage}</span>
```
Comment (josephperrott) [inline bazel/api-gen/rendering/templates/class-method-info.tsx:40]: Maybe I am misunderstanding, but are we not including the deprecation message anymore?

Comment (JeanMeche) [reply bazel/api-gen/rendering/templates/class-method-info.tsx:40]: This one was duplicated, it's handled by the `DeprecatedLabel `.

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by josephperrott** [comment]
This PR was merged into the repository by commit b9a6e876e24456058f13418e73eb426a35504999.

**Pull request closed without merging**